pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -354,9 +356,15 @@ func (r *hibernationReconciler) Reconcile(ctx context.Context, request reconcile
 		return r.checkClusterStopped(cd, false, cdLog)
 	}
 	// If we get here, we're not supposed to be hibernating
-	if isFakeCluster {
-		changed := r.setCDCondition(cd, hivev1.ClusterHibernatingCondition, hivev1.HibernatingReasonResumingOrRunning,
-			clusterResumingOrRunningMsg, corev1.ConditionFalse, cdLog)
+	if isFakeCluster || isOnpremCustomized {
```

Comment:
Why this change?

Customization shouldn't be related to hibernation support.

If the CD doesn't support hibernation, we should have caught it [above](https://github.com/openshift/hive/blob/ea63cac7a85032083993849513e53ffe65b3fe83/pkg/controller/hibernation/hibernation_controller.go#L205).

And under no circumstances should we be faking hibernate/resume for a cluster that isn't fake.

Reply:
I agree with your statement, but it didn't work and I couldn't figure it out. Thers is no actuator for Openstack but hibernation and other controllers were going crazy, the CD status was not getting to ready state. I will give it another try to debug after the current pending changes.

Reply:
Sounds good. Let me know if you're still stuck and I (or maybe @suhanime, our hibernation controller expert) can jump in and help.